templates/blink/intent_to_implement.html

```diff
@@ -24,14 +24,19 @@
   {{feature.name}}
 </div>
 
+{% comment %}
+  TODO(jrobbins): rather than checking intent_stage_int, that logic should be
+  in python code and passed to this template as a list of field names to show.
+{% endcomment %}
+
 <p>Body</p>
 <div class="email"><label>Contact emails</label>
 {% if not feature.owner %}None{% endif %}{% for owner in feature.owner %}<a href="mailto:{{owner}}">{% if forloop.last %}{{owner}}</a>{% else %}{{owner}}</a>,{% endif %}{% endfor %}
 
 <label>Explainer</label>
 {% for link in feature.explainer_links %}<a href="{{link}}">{{link}}</a>
 {% endfor %}
-{% if feature.intent_stage == "Ship" %}<label>Spec</label>
+{% if feature.intent_stage_int == 5 %}<label>Spec</label>
```

Comment:
I get why you got rid of the string. Could you do this with constants, though? That would make it easier to work out what code is doing?

Reply:
Referencing the INTENT_SHIP var (or a value derived from it) would be ideal for the sake of maintainability (understand that plumbing those into the Django content/ model object may be tricky).

Barring that, having an explanation of what 5 means in the form of a comment, since that info is lost w/ this edit.

Reply:
Perhaps the models.INTENT_STAGES object could be injected into the context map so that those values can be referenced.

https://github.com/GoogleChrome/chromium-dashboard/blob/aacad37c9269df336de7a5b966cffdfa52212c71/guide.py